### ISSUE TITLE
Wrong schematics

### ISSUE THREAD

**Issue by ozlowown** [luni64/EncPlex#1]
https://raw.githubusercontent.com/luni64/EncPlex/master/extras/Boards/MPX_4067/Schematic_part.jpg

In this pic, "EN" is connected to VCC and GND.
(i'm sorry, i don't know how to use Github :/ )

**Comment by luni64** [comment]
That's true indeed, I checked the breadboard setup which I happen to have in front of me, which connects /EN to VCC as it should be. 

> (i'm sorry, i don't know how to use Github :/ )

Nothing you can do from your side here. I'll update the schematic.

**Comment by luni64** [comment]
Fixed

**Comment by luni64** [comment]
I'm currently waiting for some  boards from china. As soon as I have them I'll continue work on the library. At the moment it is working but not yet complete...

**Comment by ozlowown** [comment]
I'll wait for news I guess! I'd wish to use your library to mux 6 encoders with switches on a Teensy 4.0. Did I miss your email address somewhere to continue this chat in private? Also I didn't find anything about your library in the pjrc forum, but I guess because it's a new project.

**Comment by ozlowown** [comment]
Also, in the same pic the encoder pin C, it's connected like B should be connected. This could be a labeling mismatch between me and you, but I've seen the "A" and "B" contact as side contacts and "C" as central.
